<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>é’»æœºé‡‡é›†æ§åˆ¶ç³»ç»Ÿ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <!-- ä¾§è¾¹æ  - é¡µé¢å¯¼èˆª -->
     <widget class="QListWidget" name="listWidget_pages">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
        QListWidget {
         background-color: #2c3e50;
         color: white;
         font-size: 14px;
         border: none;
        }
        QListWidget::item {
         padding: 15px;
         border-bottom: 1px solid #34495e;
        }
        QListWidget::item:selected {
         background-color: #3498db;
        }
        QListWidget::item:hover {
         background-color: #34495e;
        }
       </string>
      </property>
      <item>
       <property name="text">
        <string>ğŸ“Š æ•°æ®é‡‡é›†</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ğŸ® é’»æœºæ§åˆ¶</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ğŸ’¾ æ•°æ®åº“ç®¡ç†</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <!-- ä¸»å†…å®¹åŒº - StackedWidget -->
     <widget class="QStackedWidget" name="stackedWidget_pages">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <!-- æ•°æ®é‡‡é›†é¡µé¢ -->
      <widget class="QWidget" name="page_dataCollection">
       <layout class="QVBoxLayout" name="verticalLayout_1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="SensorPage" name="sensorPage"/>
        </item>
       </layout>
      </widget>
      <!-- é’»æœºæ§åˆ¶é¡µé¢ -->
      <widget class="QWidget" name="page_drillControl">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_page2">
          <property name="text">
           <string>é’»æœºæ§åˆ¶é¡µé¢ - å¾…å®ç°</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <!-- æ•°æ®åº“ç®¡ç†é¡µé¢ -->
      <widget class="QWidget" name="page_database">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_page3">
          <property name="text">
           <string>æ•°æ®åº“ç®¡ç†é¡µé¢ - å¾…å®ç°</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>æ–‡ä»¶</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>å¸®åŠ©</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_exit">
   <property name="text">
    <string>é€€å‡º</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>å…³äº</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SensorPage</class>
   <extends>QWidget</extends>
   <header>ui/SensorPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
